sql/schema_changer.go

```diff
@@ -340,6 +341,7 @@ func (sc *SchemaChanger) purgeMutations(lease *TableDescriptor_SchemaChangeLease
 				// mutations if they have the mutation ID we're looking for.
 				break
 			}
+			log.Warning("Purging schema change mutation: %v", desc.Mutations[i])
```

Comment:
good point. Will fix that and merge it. Thanks

On Tue, Jan 12, 2016 at 3:31 PM Tamir Duberstein <notifications@github.com>
wrote:

> In sql/schema_changer.go
> <https://github.com/cockroachdb/cockroach/pull/3789#discussion_r49508962>:
>
> > @@ -340,6 +341,7 @@ func (sc *SchemaChanger) purgeMutations(lease *TableDescriptor_SchemaChangeLease
> >  				// mutations if they have the mutation ID we're looking for.
> >  				break
> >  			}
> > +			log.Warning("Purging schema change mutation: %v", desc.Mutations[i])
>
> needs to be Warningf
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/3789/files#r49508962>.
>